src/main.js

```diff
@@ -356,9 +356,9 @@ class MiniGraphCard extends LitElement {
 
     if (show_legend_state) {
       legend += ` (${this.computeState(state)}`;
-      if (state !== 'unavailable') {
+      if (!(['unavailable'].include(state))) {
```

Comment:
```suggestion
      if (!(['unavailable'].includes(state))) {
```

Isn't it? Can you check if it works? I don't have access to my system currently...